src/flux/media.coffee

```diff
@@ -2,6 +2,26 @@ _ = require 'underscore'
 htmlparser = require 'htmlparser2'
 {makeSimpleStore} = require './helpers'
 
+LINKS_BEGIN = ['#']
+LINKS_CONTAIN = ['/contents/', '.cnx.org']
```

Comment:
Any reason this isn't `['.cnx.org/contents/' ]` ? That way it doesn't try to fetch `http://www.cnx.org/search` or `http://jsmith.org/blog/contents/2016-01/`